eventlet/hubs/hub.py

```diff
@@ -21,9 +22,20 @@ def alarm_signal(seconds):
 
 from eventlet.hubs import timer, IOClosed
 from eventlet.support import greenlets as greenlet, clear_sys_exc_info
-import monotonic
+from eventlet import patcher
 import six
 
+mod = os.environ.get('EVENTLET_CLOCK')
+if not mod:
+    time = patcher.original('time')
+    eventlet_clock = getattr(time, 'monotonic') if hasattr(time, 'monotonic')\
+        else getattr(patcher.original('monotonic'), 'monotonic')
+    del time
+else:
+    mod = mod.rsplit('.', 1)
+    eventlet_clock = getattr(patcher.original(mod[0]), mod[1])
```

Comment:
There is no need in original here either.

Reply:
Should it than be import_patched?
or do you mean something else?
 it is about importing user's original

Reply:
I mean should be regular Python import.

Reply:
that is good for a switch case for known options, 
on current, user's options with __import__ remain dynamic